modules/installation-creating-aws-control-plane.adoc

```diff
@@ -144,32 +144,12 @@ directory. This value is the long string with the format
 the CloudFormation template for the security group and roles.
 <21> The type of AWS instance to use for the control plane machines.
 <22> Allowed values:
```

Comment:
@kalexand-rh  please refer to values in upi/aws/cloudformation/06_cluster_worker_node.yaml in https://github.com/openshift/installer/pull/4327/files

Reply:
@yunjiang29, are you going to test them all? @sferich888 asked that only the ones that QE's going to test be added to the docs, since those are the ones that will be considered supported.

Reply:
@kalexand-rh it was a misunderstanding. It is clear now.


A minor update is: please remove `c4.2xlarge` instance type from compute node.
the current plan is:

master node: 
  m5.xlarge
  m5a.xlarge
  c4.2xlarge
  c5a.xlarge
  r5.xlarge
  r5a.xlarge

worker node: 
  m4.2xlarge
  c5.large
  r4.large
  t3.xlarge
  t3a.xlarge

I will also update it in BZ comment, sorry for the confused info, thanks.